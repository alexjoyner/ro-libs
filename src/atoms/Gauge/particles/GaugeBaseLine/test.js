import React from 'react';
import { shallow } from 'enzyme';
import { GaugeBaseLine } from './';

describe('GaugeBaseLine component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<GaugeBaseLine />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
