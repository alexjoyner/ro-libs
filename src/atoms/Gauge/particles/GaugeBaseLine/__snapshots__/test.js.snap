// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeBaseLine component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "GaugeBaseLine-uslt8t-0",
        "isStatic": true,
        "rules": Array [
          "stroke:#34495e;",
        ],
      },
      "displayName": "GaugeBaseLine",
      "render": [Function],
      "styledComponentId": "GaugeBaseLine-uslt8t-0",
      "target": "line",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
